pyproject.toml

```
@@ -34,70 +34,70 @@ test = ["pytest"]
 
 [tool.poetry.dependencies]
 case-converter = "^1.1.0"
-cookiecutter = "^2.1.1"
-cron-validator = "^1.0.6"
+cookiecutter = "^2.4.0"
+cron-validator = "^1.0.8"
 dirhash = "^0.2.1"
-email-validator = "^1.3.0"
-emoji = "^1.7.0"
-gitpython = "^3.1.31"
-google-api-core = "^2.11.0"
-google-cloud-aiplatform = "^1.22.1"
-google-cloud-build = "^3.13.0"
-google-cloud-compute = "^1.10.1"
-google-cloud-functions = "^1.11.0"
-google-cloud-logging = "^3.5.0"
-google-cloud-monitoring = "^2.14.1"
-google-cloud-notebooks = "^1.6.1"
-google-cloud-pipeline-components = "^1.0.39"
-google-cloud-resource-manager = "^1.9.0"
-google-cloud-scheduler = "^2.10.0"
-google-cloud-storage = "^2.7.0"
+email-validator = "^2.1.0"
+emoji = "^2.8.0"
+gitpython = "^3.1.40"
+google-api-core = "^2.12.0"
+google-cloud-aiplatform = "^1.35.0"
+google-cloud-build = "^3.20.1"
+google-cloud-compute = "^1.14.1"
+google-cloud-functions = "^1.13.3"
+google-cloud-logging = "^3.8.0"
+google-cloud-monitoring = "^2.16.0"
+google-cloud-notebooks = "^1.8.1"
+google-cloud-pipeline-components = "^2.5.0"
+google-cloud-resource-manager = "^1.10.4"
+google-cloud-scheduler = "^2.11.2"
+google-cloud-storage = "^2.12.0"
 gcloud-config-helper = "^0.3.1"
 halo = "^0.0.31"
 Jinja2 = "^3.1.2"
-kfp = "^1.8.19"
-pathvalidate = "^2.5.2"
+kfp = "^2.3.0"
+pathvalidate = "^3.2.0"
 pydantic = "^1.10.5"
```

Could you please allow also pydantic 2?
In the similar manner as I have it in [wanna-deploy](https://git.int.avast.com/aiops/wanna-deploy/blob/master/pyproject.toml#L18).
```suggestion
pydantic = "^1.10.10 | ^2.0.0"
```
or is there some stuff that forbids us from allowing both versions?

---

wanna-ml is very much integrated into Pedantic ecosystem and moving to 2.x just totally breaks wanna-ml.

I will create an issue and proceed with upgrade in another PR. It's a huge change unless we go v1 compatibility via the `v1.` import on the v2 pedantic. Not ideal.